apps/avatax/vitest.workspace.ts

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable import/no-default-export */
+import { defineWorkspace } from "vitest/config";
+
+export default defineWorkspace([
+  {
+    extends: "./vitest.config.ts",
+    test: {
+      name: "react",
+      exclude: ["integration/**/*.spec.ts"],
+    },
+  },
+  {
+    test: {
+      include: ["integration/**/*.spec.ts"],
+      setupFiles: ["./integration/setup.ts"],
+      name: "integration",
+      environment: "node",
+      testTimeout: 20_000, // sync webhook timeout is 20s
```

Comment:
Can we use a global timeout for requests from here as well? I saw you used it in here https://github.com/saleor/apps/blob/90eba0aaf351401b5339963364bdc3531c5a1c0d/apps/avatax/integration/setup.ts#L7

Reply:
Same with baseURL?

Reply:
Not sure what you meant by global timeout? I've added better description for the value and made baseURL into an env variable